i installed the extention and it broke my Automatic 1111

Already up to date.
venv "Z:\sd\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: v1.2.0
Commit hash: b08500cec8a791ef20082628b49b17df833f5dda
Traceback (most recent call last):
  File "Z:\sd\stable-diffusion-webui\launch.py", line 369, in <module>
    prepare_environment()
  File "Z:\sd\stable-diffusion-webui\launch.py", line 274, in prepare_environment
    run_python("import torch; assert torch.cuda.is_available(), 'Torch is not able to use GPU; add --skip-torch-cuda-test to COMMANDLINE_ARGS variable to disable this check'")
  File "Z:\sd\stable-diffusion-webui\launch.py", line 135, in run_python
    return run(f'"{python}" -c "{code}"', desc, errdesc)
  File "Z:\sd\stable-diffusion-webui\launch.py", line 111, in run
    raise RuntimeError(message)
RuntimeError: Error running command.
Command: "Z:\sd\stable-diffusion-webui\venv\Scripts\python.exe" -c "import torch; assert torch.cuda.is_available(), 'Torch is not able to use GPU; add --skip-torch-cuda-test to COMMANDLINE_ARGS variable to disable this check'"
Error code: 1
stdout: <empty>
stderr: Traceback (most recent call last):
  File "<string>", line 1, in <module>
AssertionError: Torch is not able to use GPU; add --skip-torch-cuda-test to COMMANDLINE_ARGS variable to disable this check

after installing and restarting the automatic 1111, I run the bat, got errors, decided to delete rifussion extension folder, then I tried running the WebUi and got above error, any clue why?

--- a/install.py
+++ b/install.py
@@ -3,6 +3,7 @@ import sys
 import platform
 import torch
 import launch
+import pkg_resources
 
 req_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "requirements.txt")
 
@@ -11,27 +12,20 @@ riffusion_skip_install = os.environ.get("RIFFUSION_SKIP_INSTALL", False)
 
 if not riffusion_skip_install:
     name = "Riffusion"
-    if not launch.is_installed("torchaudio"):
-        if platform.system() == "Darwin" or launch.is_installed("torch_directml"):
-            # MacOS and DirectML
-            launch.run(
-                f'"{sys.executable}" -m pip install torchaudio==0.13.1',
-                f"[{name}] Installing torchaudio...",
-                f"[{name}] Couldn't install torchaudio.",
-            )
-        elif torch.version.hip:
-            launch.run(
-                f'"{sys.executable}" -m pip install torchaudio==0.13.1+rocm5.2 --extra-index-url https://download.pytorch.org/whl/rocm5.2',
-                f"[{name}] Installing torchaudio...",
-                f"[{name}] Couldn't install torchaudio.",
-            )
-        else:
-            launch.run(
-                f'"{sys.executable}" -m pip install torchaudio==0.13.1+cu117 --extra-index-url https://download.pytorch.org/whl/cu117',
-                f"[{name}] Installing torchaudio...",
-                f"[{name}] Couldn't install torchaudio.",
-            )
+    
+    # Check if torchaudio is already installed
+    try:
+        dist = pkg_resources.get_distribution('torchaudio')
+        print(f"{name} torchaudio version: {dist.version} is already installed.")
+    except pkg_resources.DistributionNotFound:
+        print(f"{name} torchaudio is not installed, installing...")
+        launch.run(
+            f'"{sys.executable}" -m pip install torchaudio',
+            f"[{name}] Installing torchaudio...",
+            f"[{name}] Couldn't install torchaudio.",
+        )
 
+    # Install other requirements
     launch.run(
         f'"{sys.executable}" -m pip install -r "{req_file}"',
         f"[{name}] Installing requirements...",

